src/libbson/tests/json/bson_corpus/datetime.json

```diff
@@ -25,6 +25,12 @@
             "description" : "Y10K",
             "canonical_bson" : "1000000009610000DC1FD277E6000000",
             "canonical_extjson" : "{\"a\":{\"$date\":{\"$numberLong\":\"253402300800000\"}}}"
+        },
+        {
+            "description": "leading zero ms",
```

Comment:
This spec test was introduced by @kevinAlbs in https://github.com/mongodb/specifications/commit/b093e4d9d8eaca421a55308a10222907e8934c8e, but there appears to be no upstream DRIVERS ticket that would have instructed drivers to sync it.

Reply:
Ah good catch, that should have had a DRIVERS ticket created at the time (as evidenced by how long it took to discover).